Sources/EdgeBridge.swift

```diff
@@ -47,6 +57,25 @@ public class EdgeBridge: NSObject, Extension {
         return true
     }
 
+    private func handleCaptureContextData(event: Event) {
+        // add to existing list
+        contextDataCapturer.addEvent(event: event)
```

Comment:
nit: we can use `append(event: event)` or `add(event: event)`. Just improving the readility of the function.

Reply:
Yes that is fine